### PR TITLE
MK: Metric Kernel - Risk Adjusted Loss function, Order Cooldown, Fees V3

### PR DESCRIPTION
**Pull request by trdougherty**
### Description
Modifies the scoring functions to use a combination of a risk-adjusted measure and traditional returns. Updates the scoring functions needed for this, as well as hyperparameters in vali_config, and adds a number of metrics to generate_request_output for tracking.

Also updates the number of positions required to be 5 for a miner, establishes a baseline of total time on all positions, and removes the existing 3 std.dev filtering.

### Related Issues (JIRA)
[Reference any related issues or tasks that this pull request addresses or closes.]

### Checklist
- [ ] I have tested my changes on testnet.
- [*] I have updated any necessary documentation.
- [*] I have added unit tests for my changes (if applicable).
- [*] If there are breaking changes for validators, I have (or will) notify the community in Discord of the release.

### Reviewer Instructions
[Provide any specific instructions or areas you would like the reviewer to focus on.]

### Definition of Done
- [ ] Code has been reviewed.
- [ ] All checks and tests pass.
- [ ] Documentation is up to date.
- [ ] Approved by at least one reviewer.

### Checklist (for the reviewer)
- [ ] Code follows project conventions.
- [ ] Code is well-documented.
- [ ] Changes are necessary and align with the project's goals.
- [ ] No breaking changes introduced.

### Optional: Deploy Notes
[Any instructions or notes related to deployment, if applicable.]

/cc @mention_reviewer

### CODE REVIEW

File: vali_config.py [hunk 1]
```diff
@@ -146,8 +149,18 @@ class ValiConfig:
     SET_WEIGHT_LOOKBACK_RANGE_MS = SET_WEIGHT_LOOKBACK_RANGE_DAYS * 24 * 60 * 60 * 1000
 
     HISTORICAL_DECAY_TIME_INTENSITY_COEFFICIENT = 0.35
-
-    SET_WEIGHT_MINIMUM_POSITIONS = 3
+    ANNUAL_RISK_FREE_RATE = 0.02
+    DAYS_IN_YEAR = 365.25
```
Comment (jbonilla-tao) [inline vali_config.py:153]: first time i see someone take leap years into account! Mind blown
